vault/resource_identity_group_member_group_ids.go

```diff
@@ -0,0 +1,149 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/identity/group"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+)
+
+func identityGroupMemberGroupIdsResource() *schema.Resource {
+	return &schema.Resource{
+		Create: identityGroupMemberGroupIdsUpdate,
```

Comment:
Probably should be using the non deprecated CRUD funcs with this new resource.

Reply:
Whoops! Fixed in f4f6f03